etl/src/pipeline.rs

```diff
@@ -96,6 +99,18 @@ where
         // We create the table sync workers pool to manage all table sync workers in a central place.
         let pool = TableSyncWorkerPool::new();
 
+        // We create the signal used to notify the apply worker that it should force syncing tables.
```

Comment:
The pipeline initialization code is becoming cumbersome but I see no better ways of doing it.